easybuild/easyconfigs/c/CMake/CMake-3.12.1-intel-2018a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.12.1'
+
+homepage = 'http://www.cmake.org'
+
+description = """
+ CMake, the cross-platform, open-source build system.  CMake is a family of
+ tools designed to build, test and package software.
+"""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
```

Comment:
@Falkor Adding an easyconfig with both `GCCcore/6.4.0` & `intel/2018a` doesn't make much sense... When `intel/2018a` is used, the `CMake` easyconfig with `GCCcore/6.4.0` will be considered to resolve that dependency (automatically, no need for `--minimal-toolchains`, which just reverses the order).
So you can just remove this one?

Reply:
Done in d3fff9b50